interface/wx/debugrpt.h

```diff
@@ -221,8 +225,8 @@ class wxDebugReport
         is copied to a file in the debug report directory with the same name.
         Otherwise the file will be searched in the temporary directory returned
         by GetDirectory().
-        
-        The argument @a description only exists to be displayed to the user in 
+
```

Comment:
If possible, please avoid making whitespace-only changes in combination with other significant changes, this makes history more confusing.

If you really want to, please submit a separate PR removing the trailing whitespace.